setup.py

```diff
@@ -104,7 +104,7 @@ def run(self):
           "docker",
           "pymongo",
           "gdown",
-          "htcondor"
+#          "htcondor"
```

Comment:
We need to work out a way to make this optional? Can a dependency be made to fail silently?

Reply:
good question about failing silently. I'm not sure about that however we can have optional dependencies. The problem however is that it's easy to optionally have an extra dependency but not sure how to optionally have less. I'll have to have a look

Reply:
The condition added in [setup.py](https://github.com/kh296/ganga/blob/9df995563543e7dc9da3f5ff8e61268fbfda2d1f/setup.py#L96) includes `htcondor` in the dependencies except for MacOS (for which `htcondor` isn't available on pypi):

```
if sys.platform != 'darwin':
      install_requires.append("htcondor")
```

There's probably a better way of doing this, and any solution that allows for installing ganga without requiring `htcondor` would work for me :)

Reply:
I agree with @mesmith75 that the default should be to include it since it's in GangaCore and I'm happy with the suggested fix to `setup.py` as long as it works installing from PyPI